README.md

```diff
@@ -147,7 +166,10 @@ Some action expression examples:
 << ast.TRUE, nil >>
 ```
 
-Contants, functions, etc. that are returned or called should be programmed by the user in his ast (Abstract Syntax Tree) package. The ast package requires that you define your own Attrib interface as shown above. All parameters passed to functions will be of this type.
+Constants, functions, etc. that are returned or called should be programmed by 
```

Comment:
I don't know if these and most other newlines are necessary?

Reply:
It makes the file easier to edit, the rendered result is the same